libnopegl/src/deserialize.c

```diff
@@ -191,7 +191,7 @@ static int parse_kvs(const char *s, int *nb_kvsp, char ***keysp, int **valsp)
 
 static struct ngl_node **get_abs_node(struct darray *nodes_array, int id)
 {
-    const int index = ngli_darray_count(nodes_array) - id - 1;
+    const int index = (int)ngli_darray_count(nodes_array) - id - 1;
```

Comment:
Instead of casting to int, can we declare a variable `size_t count`, adapt the check below and compute the index as a size_t later on ?

Reply:
It's actually done later on in the "deserialize: parse list-based parameters using size_t". This is a temporary state because this change is actually involving the whole parser: the "node indexes" (referring to lines in the file) are typically read from the stream (as `int` at this point in the history).